subsys/bluetooth/host/Kconfig

```diff
@@ -177,6 +177,11 @@ config BT_SETTINGS_CCC_STORE_ON_WRITE
 	  workqueue stack space.
 endif # BT_SETTINGS
 
+config BT_WHITELIST
+	bool "Enable whitelist support"
+	help
+	  Enable whitelist support.
```

Comment:
Do we need help here (it provides no new information) ?

Reply:
Added some more information.